.github/workflows/cf_test_full.yml

```diff
@@ -0,0 +1,96 @@
+name: "Playwright for Cloudflare - Full Test Suite"
+
+on: [push]
```

Comment:
We need to change this to another event, maybe scheduled or something, and only for some branches